### DISCUSSION TITLE
Hacktoberfest?

### DISCUSSION THREAD

**Discussion by aloker** [sveltejs/kit#7100] [General]
A simple question: will the repo participate in [Hacktoberfest](https://hacktoberfest.com/)?

**Comment by dominikg** [comment]
No

**Reply by aloker** [reply]
Ok, thanks!

**Comment by Anudeep-313** [comment]
> A simple question: will the repo participate in [Hacktoberfest](https://hacktoberfest.com/)?

It'll be counted as valid repo only if it follows the hacktoberfest participation rules : [https://hacktoberfest.com/participation/#maintainers](https://hacktoberfest.com/participation/#maintainers)

**Reply by aloker** [reply]
That's why I asked. But if the maintainers choose not to participate, that's fine, too.

**Reply by Anudeep-313** [reply]
Yeah that's totally fine!!
